components/ContentEditor/components/FillInTheBlankModal.tsx

```diff
@@ -0,0 +1,251 @@
+import {
+    Box,
+    Button,
+    Checkbox,
+    Dialog,
+    DialogActions,
+    DialogTitle,
+    FormControlLabel,
+    FormGroup,
+} from "@mui/material";
+import { Info } from "phosphor-react";
+import React, { useState, useRef, useEffect } from "react";
+import { MuiChipsInput } from "mui-chips-input";
+import Tooltip from '@mui/material/Tooltip';
+import { blankAnswer } from "./exportStructures";
+
+export const FillInTheBlankModal = ({
+    open,
+    setOpen,
+    insert,
+}: {
+    open: boolean;
+    setOpen: (open: boolean) => void;
+    insert: (
+        e: any,
+        questionSegments: string[],
+        correctAnswers: blankAnswer[],
+    ) => void;
+}) => {
+    const toolTipMessage = "Press Enter to add possible answer choice(s).\n“Require exact match” toggles case-sensitivity and will not approximate decimals.";
+    const exactMatchText = "Require exact match";
+    // Unicode characters used to denote answer blanks when creating a FITB question.
+    // Note: Simply add more characters to placeholderChars[] array to allow for more blanks to be inserted per question! (i.e. 4 elements = max of 4 "blanks")
+    const placeholderChars = ['Ⓐ', 'Ⓑ', 'Ⓒ', 'Ⓓ'];
+    // A regular expression to help deal with placeholder-related logic.
+    const placeholderRegExp = new RegExp(placeholderChars.join('|'));
+
+    const [question, setQuestion] = useState("");
+    const [questionSegments, setQuestionSegments] = useState<string[]>([]);
+    const [correctAnswers, setCorrectAnswers] = useState<blankAnswer[]>([]);
+    const [placeholderCharCount, setPlaceholderCharCount] = useState(0);
+    const [isAddQuestionDisabled, setIsAddQuestionDisabled] = useState(true);
```

Comment:
I would remove this from a useState, and instead just have a function that determines if the add button is disabled. What that might look like:

```typescript
const addDisabled = () => {
    if (question === "" || correctAnswers.length === 0) {
      return true;
    }
    for (const correctAnswer of correctAnswers) {
      if (correctAnswer.possibleChoices[0] === undefined) {
        return true;
      }
    }
    return false;
  };
```